### PR TITLE
Updating Host Risk Score docs

### PR DESCRIPTION
**Pull request by ajosh0504**
This PR updates the Host Risk Score docs based on changes made for 8.1.

### CODE REVIEW

**Comment by ajosh0504** [comment]
@machadoum Could you also add appropriate documentation about changes you've made for 8.1, in the [#enable-kibana](https://github.com/elastic/detection-rules/blob/host_risk_score_3.0/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features) section?

**Comment by ajosh0504** [comment]
Question probably for you @brokensound77: The changes in this PR are for 8.1+. Is there a way to indicate that?

**Comment by machadoum** [comment]
> @machadoum Could you also add appropriate documentation about changes you've made for 8.1, in the [#enable-kibana](https://github.com/elastic/detection-rules/blob/host_risk_score_3.0/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features) section?

Just did it, could you check?

File: docs/experimental-machine-learning/host-risk-score.md [hunk 1]
```diff
@@ -185,8 +182,29 @@ xpack.securitySolution.enableExperimental: ['riskyHostsEnabled']
...
+![Host Risk Score card](./images/0a_host_risk_score_card.png)
+
+_Host Risk Keyword on Alert Details Flyout_
+![Host Risk Keyword](./images/0b_alert_summary.png)
+
+_Host risk classification column in the All hosts table on the Hosts page_
+![Hosts page risk classification column](./images/0c_host_page_risk_column.png)
+
+_Hosts by risk tab on the Hosts page_
+![Hosts Risk Tab](./images/0d_host_page_hosts_by_risk_tab.png)
```
Comment (ajosh0504) [inline docs/experimental-machine-learning/host-risk-score.md:198]: Can we please highlight the column, for consistency?

File: docs/experimental-machine-learning/host-risk-score.md [hunk 2]
```diff
@@ -185,8 +182,29 @@ xpack.securitySolution.enableExperimental: ['riskyHostsEnabled']
...
+
+_Host risk overview on the Host details page_
+![Host risk overview](./images/0e_host_details_page_risk_overview.png)
+
+_Hosts by risk tab on the Host details page_
+![Host Details Risk Tab](./images/0f_host_details_page_hosts_by_risk_tab.png)
```
Comment (ajosh0504) [inline docs/experimental-machine-learning/host-risk-score.md:204]: Can we highlight the two visualizations here?

**Comment by brokensound77** [comment]
> Question probably for you @brokensound77: The changes in this PR are for 8.1+. Is there a way to indicate that? Remove the `backport: auto` label perhaps?

The state of the repo is consistent across backported branches, with the exception of rule limited updates (based on `min_stack_version`). So, the docs over these branches are synced. The best thing to do is to just annotate it within these updates in a way that makes sense.

**Comment by ajosh0504** [comment]
Makes sense, @brokensound77 . I'll add Stack version info in the document itself wherever necessary.

**Review by stephmilovic — APPROVED** [review]
LGTM, let's make sure that the `ML-HostRiskScore-20220207-5.zip` or whatever the latest transform is gets uploaded before this gets merged as a new script is introduced (`ml_hostriskscore_init_script`)

**Review by brokensound77 — APPROVED** [review]
👌 

I am digging all of the screenshots 👍

**Pull request merged**